cmd/helm/flags.go

```diff
@@ -143,6 +144,9 @@ func (p *postRendererString) Set(val string) error {
 	if val == "" {
 		return nil
 	}
+	if p.options.binaryPath != "" && p.options.binaryPath != val {
```

Comment:
Is there a reason to support multiple usages with the same flag? Providing the flag mutliple times should be an error regardless

```suggestion
	if p.options.binaryPath != "" {
```